addons/crm_iap_mine/models/crm_iap_lead_mining_request.py

```diff
@@ -291,6 +291,7 @@ def _lead_vals_from_response(self, data):
         people_data = data.get('people_data')
         lead_vals = self.env['crm.iap.lead.helpers'].lead_vals_from_response(self.lead_type, self.team_id.id, self.tag_ids.ids, self.user_id.id, company_data, people_data)
         lead_vals['lead_mining_request_id'] = self.id
+        lead_vals['name'] = _("%s's opportunity", lead_vals['name'])
```

Comment:
Why ont updating in crm.iap.lead.helpers lead_vals_from_response directly ? Seems strange to have a method generating lead values, then updating then in a direct call :) .